### ISSUE TITLE
Icognito Mode Support

### ISSUE THREAD

**Issue by euricstone** [bogenpirat/remote-torrent-adder#181]
Great program. It does not work in Chrome Icognito Mode. Throws 404 error (works correctly in non-icognito tab)

Chrome Version : 
Version 55.0.2883.75 unknown-m
Google Chrome is up to date.

**Comment by bogenpirat** [comment]
got some more info about your config? using RTA in an incognito tab works fine for me with vuze.

**Comment by euricstone** [comment]
Yes, I suppose that would help wouldn't it? I'm using ruTorrent Web UI with SSL Port 443. Label/Directory are both unset for right now, Label/Directory Interactivity & Add Torrents Paused are also both unchecked.

**Comment by bogenpirat** [comment]
alright, finally managed to cobble together a test vm for rutorrent with a valid ssl certificate. and sadly/fortunately, everything works as expected in incognito mode.

would you mind typing this into your browser, replacing the respective placeholders with your config values:

https://{host}:{port}/{relativepath}/php/addtorrent.php

i get something like this:

`noty(""+theUILang.addTorrentFailed,"error");`

if you get something else, then there might be something wrong with your configuration.

**Comment by euricstone** [comment]
https://lw701.ultraseedbox.com/~merciless1/rutorrent/php/addtorrent.php

Result: 
noty(""+theUILang.addTorrentFailed,"error");

Also returned url as :
https://lw701.ultraseedbox.com/~merciless1/rutorrent/php/addtorrent.php?result[]=Failed&

Re-ran the above in non-incognito tab and everything happened exactly the same.

**Comment by euricstone** [comment]
If it works for you, it may be an issue with my own computer then. Maybe someone else can give this a try and see; I have re-tried several times though and it is not working for me.
